cybersyn/scripts/central-planning.lua

```diff
@@ -335,6 +365,11 @@ local function tick_dispatch(map_data, mod_settings)
 				goto continue
 			end
 
+			-- Verify station validity
+			if (not station.entity_stop) or (not station.entity_stop.valid) then
+				goto continue
```

Comment:
I haven't been able to trigger this branch in testing, however there was a dire warning from a previous author (presumably Mami?) saying that there is a possibility of stations being invalidated from outside this loop. I took their word for it, since I need the station to be valid to get its surface without causing a crash.